sdk/identity/Azure.Identity/src/IdentityClient.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Core.Pipeline.Policies;
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.Identity
+{
+    internal class IdentityClient
```

Comment:
Needs tests.

Reply:
Demanding tests for a prototype... rough audience.